conf/messages.en

```diff
@@ -13,14 +13,12 @@ start.need-to-do=What you need to do
 start.need-to-do.p1=Select ‘continue’ to sign in with any of your other Government Gateway user IDs. These will be older user IDs that you use for managing Self Assessment clients.
 
 
-start.task-list.heading=Copy across your existing Self Assessment clients to your agent services account
-start.task-list.need-to-know=What you need to know
-start.task-list.need-to-know.p1=Before you can sign your clients up for Making Tax Digital, you need to copy your existing relationship from your old Government Gateway user IDs to your new agent services account.
-start.task-list.need-to-know.p2=This means your clients will not have to authorise you again for Making Tax Digital services.
-start.task-list.need-to-know.panel=You cannot view your client lists through your agent services account.
-start.task-list.need-to-do=What you need to do
-start.task-list.need-to-do.p1=On the next screen you will see how many Self Assessment clients you have linked to your current user ID.
-start.task-list.need-to-do.p2=If you have more than one Government Gateway user ID, you will need to sign in with each user ID to copy more clients across.
```

Comment:
Are these messages no longer being used and that's why you have deleted them?

Reply:
I have added this content as per excel sheet attached with the ticket and as per slide deck.